### PR TITLE
Symfony 6 support

### PR DESCRIPTION
**Pull request by kamilmusial**
- Drop Symfony 4.4
- Drop php < 8.1

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -15,11 +15,9 @@ jobs:
       fail-fast: false
       matrix:
         symfony-version:
-          - '^4.4'
           - '^5.4'
+          - '^6.0'
         php-version:
-          - "7.4"
-          - "8.0"
           - "8.1"
```
Comment (dbu) [inline .github/workflows/ci.yaml]: could you add 8.2 as well?

**Review by dbu — COMMENTED** [review]
thanks. looks good to me - can you look into the test failure? the null vs empty array is probably equivalent in the outcome but confuses the test.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (kamilmusial) [reply .github/workflows/ci.yaml]: Sure!

Comment (kamilmusial) [reply .github/workflows/ci.yaml]: Updated

**Comment by goetas** [comment]
thank you!

**Pull request merged**